WalletWasabi.Fluent/Helpers/CoinHelpers.cs

```diff
@@ -32,4 +34,21 @@ public static SmartLabel GetLabels(this SmartCoin coin, int privateThreshold)
 	}
 
 	public static int GetConfirmations(this SmartCoin coin) => coin.Height.Type == HeightType.Chain ? (int)Services.BitcoinStore.SmartHeaderChain.TipHeight - coin.Height.Value + 1 : 0;
+
+	public static PrivacyLevel GetPrivacyLevel(this SmartCoin coin, Wallet wallet)
```

Comment:
Why not pass `AnonScoreTarget` here instead of `Wallet`?

Reply:
In general I prefer to use entire objects as opposed to individual primitive values. This leaves the responsibility of "grabbing" the actual value from the object inside the method itself, instead of leaving it to its consumers

Reply:
Yeah, this is an interesting case... 

I get your point and I would agree in most cases I guess. But it kind of sounds weird that a wallet is need to decide above a coin's privacy level. Sounds like a huge dependency. And also all the other methods in the class receive the `privateThreshold` param individually so for consistency I agree with @yahiheb in this case.

Reply:
I am happy with your solution now. Just one thing to mention:

> The setting needed to determine the coin's privacy level actually lives inside the wallet, so that dependency already exists, conceptually, regardless of how the code is written.

If you have a SmartCoin instance that doesn't mean that any of your wallets contains that.